packages/mds-agency/request-handlers.ts

```diff
@@ -218,9 +218,16 @@ export const updateVehicle = async (req: AgencyApiUpdateVehicleRequest, res: Age
   }
 }
 
-const uuidSchema = { type: 'string', format: 'uuid' }
-const numberSchema = { type: 'number' }
-const timestampSchema = { type: 'integer', minimum: 100_000_000_000, maximum: 99_999_999_999_999 }
+const uuidSchema = <const>{ type: 'string', format: 'uuid' }
```

Comment:
Is there one central place we can stick `uuidSchema` and `timestampSchema` instead of redeclaring them?

Reply:
not until the _other_ Joi-based definitions are removed. But soon.